next.config.js

```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  images: {
+    loader: 'custom',
```

Comment:
Do you need that? re cobraz/next-theme-ui#11

Reply:
If I don't have this in the config I get this error: 

```
Error: Image Optimization using Next.js' default loader is not compatible with `next export`.
  Possible solutions:
    - Use `next start` to run a server, which includes the Image Optimization API.
    - Use any provider which supports Image Optimization (like Vercel).
    - Configure a third-party loader in `next.config.js`.
    - Use the `loader` prop for `next/image`.
  Read more: https://nextjs.org/docs/messages/export-image-api
    at /Users/brage/Kode/bjerk/website/node_modules/next/dist/export/index.js:257:23
    at async Span.traceAsyncFn (/Users/brage/Kode/bjerk/website/node_modules/next/dist/trace/trace.js:75:20)
error Command failed with exit code 1.
```

Reply:
Have you fixed `next-theme-ui` to version 1.2.0?

Reply:
Just tried this. Downgraded to 1.2.0, deleted `node_modules`, `yarn.lock` and all build outputs. Still getting the same error. The next config thing seems necessary :/

Reply:
Okey, it's probably fine for now anyway. :)